README.md

```diff
@@ -120,62 +126,51 @@ Note: While the ``tune download`` command allows you to download *any* model fro
 
 #### Running recipes
 
-TorchTune contains recipes for:
+TorchTune contains built-in recipes for:
 - Full finetuning on [single device](https://github.com/pytorch/torchtune/blob/main/recipes/full_finetune_single_device.py) and on [multiple devices with FSDP](https://github.com/pytorch/torchtune/blob/main/recipes/full_finetune_distributed.py)
 - LoRA finetuning on [single device](https://github.com/pytorch/torchtune/blob/main/recipes/lora_finetune_single_device.py) and on [multiple devices with FSDP](https://github.com/pytorch/torchtune/blob/main/recipes/lora_finetune_distributed.py).
 - QLoRA finetuning on [single device](https://github.com/pytorch/torchtune/blob/main/recipes/lora_finetune_single_device.py), with a QLoRA specific [configuration](https://github.com/pytorch/torchtune/blob/main/recipes/configs/7B_qlora_single_device.yaml)
 
-To run a full finetune on two devices on the Alpaca dataset using the Llama2 7B model and FSDP:
-
+To run a LoRA finetune on a single device using the [Alpaca Dataset](https://huggingface.co/datasets/tatsu-lab/alpaca):
 ```
-tune --nnodes 1 --nproc_per_node 2 \
-full_finetune_distributed \
---config llama2/7B_full
+tune run lora_finetune_single_device --config llama2/7B_lora_single_device
 ```
 
-The argument passed to `--nproc_per_node` can be varied depending on how many GPUs you have. A full finetune can be memory-intensive, so make sure you are running on enough devices. See [this table](https://github.com/pytorch/torchtune/blob/main/README.md#finetuning-resource-requirements) for resource requirements on common hardware setups.
+TorchTune integrates with [`torchrun`](https://pytorch.org/docs/stable/elastic/run.html) for easily running distributed training. See below for an example of running a Llama2 7B full-finetune on two GPUs.
 
-Similarly, you can finetune with LoRA on the Alpaca dataset using the Llama2 13B model on two devices via the following.
+> Make sure to place any torchrun commands **before** the recipe specification b/c any other CLI args will
+overwrite the config, not affect distributed training.
 
 ```
-tune --nnodes 1 --nproc_per_node 2 \
-lora_finetune_distributed \
---config llama2/13B_lora
+tune run --nproc_per_node 2 full_finetune_distributed --config llama2/7B_full_distributed
```

Comment:
where did --nnodes go? Are we not supporting it anymore or is it just hidden?

Reply:
`--nnodes` defaults to 1 already and we don't technically support multinode anyway so removing it for now.